src/lib/accounts.js

```diff
@@ -38,3 +38,18 @@ export function getAccountsByType (cozy, accountType, accountsIndex) {
     })
   }
 }
+
+export function getAllAccounts (cozy, accountsIndex) {
+  if (accountsIndex) {
```

Comment:
Isn't the operation of defining the index supposed [to be indempotent](https://cozy.github.io/cozy-client-js/#cozyclientdatadefineindexdoctype-fields) ? Then do we need to keep the index ? The code without this would be quite simpler.